### PR TITLE
GEN-1686 - styles(ProductReviews): tweaks

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Update width
* Add space between reviews filter options
* Truncate the size of review comments by 3 at maximum

## Justify why they are needed

Requested by Peter/William

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3824** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3824?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #3825** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3825?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3824?utm_source=stack-comment).

File: packages/ui/src/components/Text/Text.tsx [hunk 1]
```diff
@@ -56,8 +56,13 @@ export const TextBase = styled(
   ...(strikethrough && { textDecorationLine: 'line-through' }),
 }))
 
-export const Text = ({ as, balance, children, className, ...rest }: TextProps) => (
-  <TextBase as={as} className={className} {...rest}>
-    {!balance ? children : <Balancer>{children}</Balancer>}
-  </TextBase>
+export const Text = forwardRef<HTMLDivElement, TextProps>(
```
Comment (gustaveen) [inline packages/ui/src/components/Text/Text.tsx:59]: nit: Hm, I wonder if we should pass ref to basic UI components. Could we maybe create a Content component for the use case instead that takes a ref? 
But maybe we will have more use cases for it. :) Does it matter that for the type `HTMLDivElement` if we make it render as `span` or `p` instead via the `as` prop?

**Review by gustaveen — APPROVED** [review]
Great job! 🥳 Just had a comment about passing ref

File: packages/ui/src/components/Text/Text.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply packages/ui/src/components/Text/Text.tsx]: Got ya. The markup would be simpler if `Text` would take a `ref` but I can change it back.

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Jan 11, 5:02 AM**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3824).

**Pull request merged**